games/minetest/Portfile

```diff
@@ -7,10 +7,11 @@ PortGroup               cmake 1.1
 compiler.cxx_standard   2011
 compiler.thread_local_storage   yes
 
+name                    minetest
```

Comment:
please remove, this is already set from the "github.setup" line

Reply:
OK